### DISCUSSION TITLE
About Student developer pack

### DISCUSSION THREAD

**Discussion by Uhanjani** [community/community#77961] [GitHub Education]
### Select Topic Area

Question

### Body

Iam a student. l got student developer pack with my college related mail-id unfortunately l have deleted that GitHub account. Is it possible to get student developer pack again with same mail-id

**Comment by leeringrose** [comment]
Unfortunately, once an account is deleted, it may not be possible to retrieve it, along with any associated perks or packs.
However, you may want to reach out to GitHub support directly to see if they can help with your specific situation.
They may be able to provide guidance or assistance for your case.
